src/Nethermind/Nethermind.Blockchain.Test/Validators/TxValidatorTests.cs

```diff
@@ -120,5 +121,28 @@ public void Before_eip_155_has_to_have_valid_chain_id_unless_overridden(bool val
             TxValidator txValidator = new TxValidator(1);
             txValidator.IsWellFormed(tx, releaseSpec).Should().Be(!validateChainId);
         }
+        
+        [TestCase(TxType.Legacy, true, ExpectedResult = true)]
+        [TestCase(TxType.Legacy, false, ExpectedResult = true)]
+        [TestCase(TxType.AccessList, false, ExpectedResult = false)]
+        [TestCase(TxType.AccessList, true, ExpectedResult = true)]
+        [TestCase((TxType)100, true, ExpectedResult = false)]
+        public bool Before_eip_2930_has_to_be_legacy_tx(TxType txType, bool eip2930)
+        {
```

Comment:
how about other types -> what I noticed you only check Legacy type but allow other types (5, 6 and so on)

Reply:
there is test for type 100